src/simulator/src/Verilog2CV.js

```diff
@@ -65,23 +65,23 @@ export function verilogModeSet(mode) {
     if (mode == verilogMode) return
     verilogMode = mode
     if (mode) {
-        $('#code-window').show()
-        $('.elementPanel').hide()
-        $('.timing-diagram-panel').hide()
-        $('.quick-btn').hide()
-        $('#verilogEditorPanel').show()
+        document.querySelector('#code-window').style.display = 'block';
+        document.querySelector('.elementPanel').style.display = 'none';
+        document.querySelector('.timing-diagram-panel').style.display = 'none';
+        document.querySelector('.quick-btn').style.display = 'none';
+        document.querySelector('#verilogEditorPanel').style.display = 'block';
```

Comment:
lets use `getElementById` for ids, since its faster then `querySelector`

Reply:
Thanks for reviewing!
I have replaced querySelector by getElementById.

Reply:
cool :100: